mne/source_estimate.py

```diff
@@ -1395,7 +1419,10 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
                                       config_opts=config_opts,
                                       subjects_dir=subjects_dir, figure=figure,
                                       views=views, colorbar=colorbar,
-                                      clim=clim)
+                                      clim=clim, cortex=cortex, size=size,
+                                      background=background,
+                                      foreground=foreground,
+                                      time_unit=time_unit)
```

Comment:
While we're at it, could there be a param for setting the time point showed (like ``set_data_time_index``). Sure, you can do it after plotting, but it doesn't seem to work too well on some settings.